chain-vote/src/math/babystep.rs

```diff
@@ -36,7 +36,11 @@ impl BabyStepsTable {
     ///
     /// For example, a balance of 2 means that the table will precompute 2 times more
     /// baby steps than the standard O(sqrt(n)), 1 means symmetrical steps.
-    pub fn generate_with_balance(max_value: u64, balance: u64) -> Self {
+    pub fn generate_with_balance(max_value: u64, balance: u64) -> Option<Self> {
```

Comment:
Or we could use `NonZerou64`

Reply:
Yeah, just using `NonZero*` types, why haven't I though about them in this place :)

Reply:
Also use the non-zero type for balance, because setting zero balance will result in the zero step size. And then we can just remove that `assert!` that was panicking.